rules/csharp/security/insecure-fspickler-deserialization.yml

```
@@ -0,0 +1,12 @@
+id: insecure-fspickler-deserialization
+severity: warning
+language: csharp
+message: Avoid using FSPickler, it is insecure and can lead to remote code execution
+note: >-
+  [CWE-502]: Deserialization of Untrusted Data
+  [OWASP A08:2017]: Insecure Deserialization
+  [OWASP A08:2021]: Software and Data Integrity Failures
+  [REFERENCES]
+      - https://mbraceproject.github.io/FsPickler/tutorial.html#Disabling-Subtype-Resolution
+rule:
+  pattern: FsPickler.CreateJsonSerializer()
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)